src/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -53,14 +53,26 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
                 netcoreSdkVersion = new DirectoryInfo(netcoreSdkDir).Name;;
             }
 
+            if (!string.IsNullOrEmpty(sdkReference.MinimumVersion) && 
+                SemanticVersion.Parse(netcoreSdkVersion) < SemanticVersion.Parse(sdkReference.MinimumVersion))
+            {
+                return factory.IndicateFailure(
+                    new[]
+                    {
+                        $"Version {netcoreSdkVersion} of the SDK is smaller than the minimum version"
+                        + $" {sdkReference.MinimumVersion} requested. Check that a recent enough .NET Core SDK is"
+                        + " installed and/or increase the version specified in global.json."
```

Comment:
" installed, increase the minimum version specified in the project, or increase the version specified in global.json"